packages/web-config/webpack.config.base.ts

```diff
@@ -0,0 +1,170 @@
+import webpack from 'webpack'
+import CleanWebpackPlugin from 'clean-webpack-plugin'
+import sassGlobImporter = require('node-sass-globbing')
+import LiveReloadPlugin from 'webpack-livereload-plugin'
+import HtmlWebpackPlugin = require('html-webpack-plugin')
+import MiniCSSExtractWebpackPlugin = require('mini-css-extract-plugin')
+import chalk from 'chalk'
+
+const env = process.env.NODE_ENV === 'production' ? 'production' : 'development'
+const args = process.argv.slice(2)
+const liveReloadEnabled = !args.includes('--no-livereload')
+const watchModeEnabled = args.includes('--watch') || args.includes('-w')
+// opt out of livereload with arg --no-livereload
+if (liveReloadEnabled && watchModeEnabled) console.log(chalk.gray(`\nLive Reloading is enabled. use ${chalk.bold('--no-livereload')} to disable`))
+process.env.NODE_ENV = env
+
+const config: webpack.Configuration = {
+	mode: 'none',
```

Comment:
What's up with the spacing here and in other files? It should be 2-space indentation to remain consistent with the rest of the codebase.

Reply:
We don't have a typescript linter yet, so could this wait for #4449 ?

Reply:
ah i see, didn't even notices I was using tabs. Fixed